docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+the operation is performed. It is used for restoring focus on the same element when the user navigates back to the same
+page.
+
+### `{inputMapper.base}.nextPageInHistory(that, value)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Navigates to the next page in history when the gamepad `button` or `trigger` is pressed. It uses the
+[`History API`](https://developer.mozilla.org/en-US/docs/Web/API/History_API) for the core functionality. The `focused
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element if the user navigates back to the same
+page.
+
+### `{inputMapper.base}.thumbstickHistoryNavigation(that, value, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `invert {Boolean}` Whether the thumbstick history navigation should be in opposite order.
+- Returns: Nothing.
+
+Calls the [`{inputMapper.base}.previousPageInHistory`](#inputmapperbasepreviouspageinhistorythat-value) and
+[`{inputMapper.base}.nextPageInHistory`](#inputmapperbasenextpageinhistorythat-value) invokers according to the
+direction the `thumbstick` is pressed. The direction can be inverted by passing `true` as the `invert` argument.
+
+### `{inputMapper.base}.reverseTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for backward tab navigation, i.e. `reverseTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous tabbable element using gamepad `buttons` and
+`triggers`. The previous tabbable element is the element before the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.forwardTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for forward tab navigation, i.e. `forwardTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the next tabbable element using gamepad `buttons` and
+`triggers`. The next tabbable element is the element next to the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.thumbstickTabbing(that, value, speedFactor, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of tab navigation should be increased.
+- `invert {Boolean}` Whether the thumbstick tab navigation should be in opposite order.
```

Comment:
Same comment here about needing to know what the default order is before you can understand the inverse.  You can use a simple single-sentence example, "for example, left on the horizontal axis of a thumbstick...".